script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -51,12 +51,12 @@ It's early in the morning and you're already swarming in and blocking the way."
 Vicious words reverberate in the classroom as if to ignore the fact that classmates are present.
 I don't need to see it to confirm it. The atmosphere in the classroom has changed because of the arrival of this―――
 <Page14>
-"Sup! Good mornin' Glasses-kun!
-You look so irritated this morning, you look like a piece of shit!"
+"Sup! Good mornin' four eyes!
+You look so irritated this morning, you look like a piece of shit!" //Tl recheck needed
```

Comment:
> 今朝も辛気くさい顔しやがって、まったくクソみたいなヤロウですねぇ！

Reply:
>You look so irritated this morning, really, you look like a piece of shit

or something around that since he said ”またく” or it think that should be written as ”全く”... like the next one after the comma is something like "really, you look like a piece of shit".. he being frank or something around that

Reply:
The reason why I asked for this to be rechecked is because "you look so irritated, you look like a piece of shit" reads like "you look so irritated THAT you look like a piece of shit" rather than it being basically two separate thoughts.
> Man you look irritated this morning. You seriously look like shit!